project/Dependencies.scala

```diff
@@ -2,12 +2,12 @@ import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
 import sbt._
 
 object Dependencies {
-
+  // scalastyle:off
   //libs for testing
   lazy val testing: Def.Initialize[Seq[ModuleID]] = Def.setting(Seq(
-    "org.scalatest" %%% "scalatest" % Versions.scalaTest % Test,
+    "org.scalatest" %%% "scalatest" % Versions.scalaTest % Test/*,
 
-    "org.scalatest" %%% "scalatest-matchers" % Versions.scalaTest % Test))
+    "org.scalatest" %%% "scalatest-matchers" % Versions.scalaTest % Test*/))
```

Comment:
Thanks Anton.I picked the version from scalatest website listed [here](http://www.scalatest.org/install). However I will update it to the latest version and update the pull request. However their [release page](https://github.com/scalatest/scalatest/releases) says latest version is 3.0.0-RC1 and not 3.0.0-M16-SNAP4. So is it okay to use this one? Also I was wondering if you plan to add some test coverage as well.